app/controllers/api/v1x0/portfolio_items_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
           collection(Tag.find(params.require(:tag_id)).portfolio_items)
         else
           authorize(PortfolioItem)
-          collection(policy_scope(PortfolioItem.all), :pre_authorized => true)
+          collection(PortfolioItem.all)
```

Comment:
@eclarizio Doing a PortfolioItem.all here is going to include all PortfolioItems but it should be all PortfolioItems that belong to readable Portfolios.

Reply:
That should be handled in the `collection` method when it goes and does the `policy_scope`.

Reply:
Looking into this question. Yes the policy gets picked up in `rbac_scope` called from `scope` which is referenced in `collection`
https://github.com/RedHatInsights/catalog-api/blob/master/app/controllers/api/v1x0/mixins/index_mixin.rb#L5-L41